src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -311,16 +319,18 @@ function MultiCombobox<Item>({
               >
                 <Flex as="ul" wrap="wrap" align="baseline" pl={3} pr={10} pt={itemsPt} pb="2px">
                   <>
-                    {value.map(selectedItem => (
-                      <Tag
-                        as="li"
-                        key={itemToString(selectedItem)}
-                        m={1}
-                        onRemove={() => removeItem(selectedItem)}
-                      >
-                        {itemToString(selectedItem)}
-                      </Tag>
-                    ))}
+                    {alwaysUsePlaceholder && Boolean(value.length)
```

Comment:
I believe that the designs are a bit different since they don't use a `label` at all. We should rethink our approach entirely, rather than adding "exception props". We have a million variations for the MultiCombobox, to the point that it's almost hard to manage. Even If we absolutely need this ASAP, then I'd suggest we add a prop that overrides the input completely rather than relying on the placeholder (which unfortunately has the wrong opacity).

Could we sync tomorrow around it during our call?

Reply:
yes.  should we loop jim in?